### ISSUE TITLE
Documentation for older versions

### ISSUE THREAD

**Issue by yasuf** [test-prof/test-prof#235]
Hey guys, I have a question about the postgresql version requirement for version 0.9.0, is there any way I can find documentation for that? I'm using version 10.3 right now and getting intermittent errors in unit tests

**Comment by palkan** [comment]
> is there any way I can find documentation for that?

Yes, you can browse the docs on GitHub for a specific tag: https://github.com/test-prof/test-prof/tree/v0.9.0/docs

> I'm using version 10.3 right now and getting intermittent errors in unit tests

10.3 should work; which parts of TestProf do you use and what kind of errors do you see?
